README.md

```diff
@@ -60,3 +61,148 @@ to the existing annotations:
+   (A clone is thus simply a set of cells.)
+7. Error-correct the clone graph by removing spurious edges ("bridges").
+
+
+## Output files
+
+`trex run` by default writes its results to a newly created directory named
+`trex_run`.
+The name of the output directory can be changed with `--output` (or `-o`).
+The files created in the output directory are described below.
+They are listed in the order in which the program creates them,
+see the "Pipeline steps overview" section for more details about what is
+done in each step.
+
+Many of the files are tables in either tab-separated value (TSV) format
+or in comma-separated values (CSV) format.
+
+
+### `log.txt`
+
+This file contains a copy of the output that `trex run10x` prints to the
+terminal.
+
+
+### `reads.txt`
+
+A table listing *cell_id*, *umi* and *clone_id* for each usable input read.
+Example:
+
+    #cell_id          umi         clone_id
+    TGACGGCGTTACCAGT  AAAAAACTGT  TGTCAATCGTTCGGTTGAGCAAGATCTTAG
+
+
+### `entries.bam`
+
+The actual reads used when creating `reads.txt`,
+copied over from the input BAM file.
+
+
+### `molecules.txt`
+
+A table listing *cell_id*, *umi* and *clone_id* for each detected molecule.
+Example:
+
+    #cell_id                umi             clone_id
+    AAACCTGAGAGGTACC        AGTTAAAGTA      TGTCAATCGTTCGGTTGAGCAAGATCTTAG
+
+
+### `molecules_corrected.txt`
+
+A table with molecules where clone IDs have been error corrected.
+
+The file is the same as `molecules.txt` with some changes in the *clone_id*
+column.
+
+
+### `cells.txt`
+
+A table listing the detected cells.
+The fields are *cell_id*, a colon (`:`) and then
+pairs of columns *clone_id1* and *count1* for each clone ID found in that cell.
+Example:
+
+    #cell_id          :  clone_id1                       count1  clone_id2 count2  ...
+    AAACCTGAGAGGTACC  :  TGTCAATCGTTCGGTTGAGCAAGATCTTAG  1
+    AAACCTGAGTAGCGGT  :  TGTCAATCGTTCGGTTGAGCAAGATCTTAG  3
+
+
+### `cells_filtered.txt`
+
+The same as `cells.txt`, but with error-corrected clone ID counts.
+Cells that end up without any clone IDs after error correction are removed.
+
+
+### `umi_count_matrix.csv`
+
+A matrix of UMI counts with cells as columns and clone IDs as rows.
+
+Only created if option `--umi-matrix` is used.
+
+
+### `components.txt`
+
+Connected components of the clone graph.
+
+
+### `components_corrected.txt`
+
+Connected components of the clone graph after error correction.
+
+
+### `graph.pdf`, `graph_corrected.pdf`
+
+Plots of the clone graph and the error-corrected clone graph.
+
+These are only created if option `--plot` is used.
+
+`graph_corrected.gv` and `graph.gv` are textual descriptions of the graphs in GraphViz format.
+
+
+### `clones.txt`
+
+A table listing the cell IDs belonging to each clone.
+The columns are *clone_id* and *cell_id* where *clone_id* is a number that
+identifies the clone.
+
+    #clone_id,cell_id
+    1,TGGCGCAAGAATAGGG
+
+
+### `clone_sequences.txt`
```

Comment:
Is a description for this one missing? If yes:
A table listing the 30N sequence of each clone ID.
The columns are *clone_id* and *clone_seq* where *clone_id* is a number that identifies the clone and *clone_seq* its nucleotide sequence.

#clone_id,clone_seq
1,ACTAGGAGATTGACGGATCACCTTTGGTCG

Reply:
Thanks, added.